vulnerable_people_form/form_pages/check_your_answers.py

```diff
@@ -33,8 +33,6 @@ def post_check_your_answers():
     registration_number = persist_answers_from_session()
     session["registration_number"] = registration_number
 
-    is_spl_match = spl_check.check_spl(form_answers()["nhs_number"], form_answers()["date_of_birth"])
-    if "NOTIFY_DISABLED" not in current_app.config:
-        govuk_notify_client.send_notification(registration_number, is_spl_match)
+    notifications.send_message(registration_number)
```

Comment:
if we no longer do the SPL check can we also remove that from the code base - e.g in the integrations folder